src/assets/drizzle/styles/components/swatch.css

```diff
@@ -0,0 +1,38 @@
+:root {
+  --Swatch-sample-stroke: rgba(0, 0, 0, 0.1);
+  --Swatch-sample-stroke-width: 2px;
+  --Swatch-sample-width: 10em;
+  --Swatch-sample-height: 5.625em;
+  --Swatch-info-padding: calc(var(--space) / 4);
+}
+
+.Swatch {
+  margin: 0;
+  display: inline-block;
+}
+
+.Swatch > * {
+  margin: 0;
+}
+
+/**
+ * 1. Double the intended stroke-width because only the inner half will be
+ *    within the visible viewBox.
+ */
+
+.Swatch-sample {
+  display: block;
+  width: var(--Swatch-sample-width);
+  height: var(--Swatch-sample-height);
+  stroke: rgba(0, 0, 0, 0.1);
```

Comment:
Also, it seems like most of the unique CSS lives within this class. I wonder if this element could be the `.Swatch` and everything else can be achieved with existing utilities.